templates/app/models/sequelize-model.hbs

```diff
@@ -20,20 +20,43 @@ module.exports = (sequelize, DataTypes) => {
     schema: process.env.DATABASE_SCHEMA,{{/if}}
   });
 {{#if noId}}  {{modelName}}.removeAttribute('id');{{/if}}
-  // This section contains the relationships for this model. See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
+  // This section contains the relationships for this model.
+  // See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
   {{modelName}}.associate = (models) => {
     {{#each references as |reference|}}
+    {{#if reference.isBelongsTo}}
     {{../modelName}}.belongsTo(models.{{reference.ref}}, {
       foreignKey: {
         name: '{{reference.foreignKeyName}}',{{#if reference.foreignKeyColumnUnconventional}}
-        field: '{{reference.foreignKey}}',{{/if}}
+        field: '{{reference.columnName}}',{{/if}}
       },{{#if reference.targetKey }}
       target: {
         name: '{{reference.targetKey}}',{{#if reference.targetKeyColumnUnconventional}}
         field: '{{reference.targetKey}}',{{/if}}
       },{{/if}}{{#if reference.as}}
       as: '{{reference.as}}',{{/if}}
     });
+    {{/if}}
+    {{#if reference.isHasOneOrHasMany}}
+    {{../modelName}}.{{reference.association}}(models.{{reference.ref}}, {
+      foreignKey: {
+        name: '{{reference.foreignKeyName}}',{{#if reference.foreignKeyColumnUnconventional}}
+        field: '{{reference.columnName}}',{{/if}}
+      },{{#if reference.targetKey }}
+      target: {
+        name: '{{reference.targetKey}}',{{#if reference.targetKeyColumnUnconventional}}
+        field: '{{reference.targetKey}}',{{/if}}
+      },{{/if}}{{#if reference.as}}
+      as: '{{reference.as}}',{{/if}}
+    });
+    {{/if}}
+    {{#if reference.isBelongsToMany}}
+    {{../modelName}}.belongsToMany(models.{{reference.junctionTable}}, {
+      through: '{{reference.ref}}',
+      foreignKey: '{{reference.columnName}}',
+      otherKey: '{{reference.otherKey}}',
+    });
+    {{/if}}
```

Comment:
the relationships should be ordered: First the belongsTo, then the hasOne the the hasMany and finaly the ManyToMany